indico/modules/events/abstracts/forms.py

```diff
@@ -200,6 +227,34 @@ def __init__(self, *args, **kwargs):
         self.merged_into.excluded_abstract_ids = {abstract.id}
 
 
+class AbstractReviewForm(IndicoForm):
+    """Form for reviewing an abstract"""
+
+    _order = ('comment', 'proposed_action', 'proposed_contribution_type', 'proposed_related_abstract')
+
+    comment = TextAreaField(_("Comments"))
```

Comment:
OTOH, I've been using `render_kw={'placeholder': _("Leave a comment...")}` to have a placeholder in the `textarea` while hiding the form labels. I think it looks nice since we don't have that many fields in this forms and the labels make them look bloated.